Dockerfile-default-cpu

```diff
@@ -31,7 +34,6 @@ ARG TORCH_TB_PROFILER_PIP
 RUN if [ "$TORCH_TB_PROFILER_PIP" ]; then pip install $TORCH_TB_PROFILER_PIP; fi
 
 ARG HOROVOD_PIP=horovod
-ARG HOROVOD_WITH_TENSORFLOW=1
```

Comment:
ideally I'd still like this to be an arg separate from `$TENSORFLOW_PIP`: e.g. if you are building an image off base tensorflow image, you aren't going to have `TENSORFLOW_PIP` set because you will not install it.

however since you'll be installing horovod, you'd want to set this flag.

in other words, it's good we can compute default ARG value based off TENSORFLOW_PIP, but then it'd be great to have a way to override it.

Reply:
Added an override ARG for this.